web/src/cypress/support/select-by-label.ts

```diff
@@ -21,7 +21,18 @@ type selectByLabelFn = (label: string) => Cypress.Chainable
 type findByLabelFn = (label: string) => Cypress.Chainable
 type multiRemoveByLabelFn = (label: string) => Cypress.Chainable
 
+function isSearchSelect(sub: any): Cypress.Chainable<boolean> {
+  return cy.wrap(sub).then(el => {
+    return el.parents('[data-cy=material-select]').data('cy') === 'material-select'
+  })
+}
+
 function selectByLabel(sub: any, label: string): Cypress.Chainable {
+  // clear value in search select
+  if ((!label || label === '{backspace}') && isSearchSelect(sub)) {
```

Comment:
I adjusted the parent function to return with the then earlier, lemme know if you believe this might have unintended effects:
```
function isSearchSelect(sub: any): Cypress.Chainable<boolean> {
  return cy.wrap(sub).then(el => {
    return el.parents('[data-cy=material-select]').data('cy') === 'material-select'
  }).then(res => res)
}
```

Reply:
This will still return a promise chain, the `.then(res=>res)` is effectively a no-op.

To use this:
```js
// instead of this
function selectByLabel(sub) {
  if (isSearchSelect(sub)) {
    // do thing
  }
}

// it would have to be used like this
function selectByLabel(sub) {
  return isSearchSelect(sub).then(isActuallySearchSelectOrNot => {
    if (isActuallySearchSelectOrNot) {
      // do thing
    }
  })
}
```

---

We can look into using `async` and `await` which would make things way cleaner, but IIRC it doesn't play well with Cypress and Typescript, might be better now as that was a while ago.